src/Iris/Tests/Tactics.lean

```diff
@@ -115,6 +115,43 @@ theorem multiple_patterns [BI PROP] (Q : PROP) : ⊢ □ (P1 ∧ P2) -∗ Q ∨
 
 end intro
 
+-- revert
+namespace revert
```

Comment:
It would also be good to have tests for the failing cases:
1. What happens when the name is not used?
2. What happens when a pure hypothesis is used in another hypothesis?

For the second point: When I try the following test,
```lean
theorem fail [BI PROP] (Φ : Bool → PROP) : ⊢ ∀ x, <affine> ⌜x = true⌝ -∗ Φ x -∗ Φ x := by
  iintro x %_ H
  irevert x
```
something very weird happens:
```lean
Goals (1)
PROP : Type u_1
inst : BI PROP
Φ : Bool → PROP
⊢ 
  ∗H : Φ _fvar.2200
  ⊢ «forall» fun x x_1 => Φ x
```